BeforeBuild.cmd

```diff
@@ -0,0 +1,32 @@
+@ECHO off
+SETLOCAL
+
+if NOT exist "GeneratedKey.snk" (
+sn -k GeneratedKey.snk
+)
+
+if "%ERRORLEVEL%" NEQ "0" (
+echo COMMAND sn -k FAILED
+goto ExitFailed
+)
+
+if NOT exist "GeneratedKey.snk" (
+goto ExitFailed
```

Comment:
Echo a message stating that the GeneratedKey.snk file could not be found. I don't think the 'if' statement will be printed to the console, so we should explicitly state why the script failed.